### PR TITLE
Adding additional required linux apt packages

### PR DESCRIPTION
**Pull request by spham-amzn**
Signed-off-by: Steve Pham <spham@amazon.com>

### CODE REVIEW

**Review by willihay — APPROVED** [review]

**Comment by sptramer** [comment]
@burelc-amzn Will merge after tech approval

File: content/docs/welcome-guide/requirements.md
Comment (burelc-amzn) [inline content/docs/welcome-guide/requirements.md:210]: Nit: maybe we should group all the xcb/xkb packages together?

**Pull request merged**
